From 8c839b9bbd3fca53d8f5a530d1a2285419cf3071 Mon Sep 17 00:00:00 2001
From: Alexey Shvetsov <alexxy@gentoo.org>
Date: Tue, 19 Mar 2013 00:48:48 +0400
Subject: [PATCH] LU-2984 build: Fix warning with gcc-4.6+ in mdt_handler.c
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

gcc-4.6+ has more stricter check for possible usage of unnitialized
variables so it will generate error with -Werror like this

lustre/mdt/mdt_handler.c: In function ‘mdt_get_info’:
lustre/mdt/mdt_handler.c:5508:9: error: ‘rc’ may be used uninitialized
in this function [-Werror=maybe-uninitialized]
cc1: all warnings being treated as errors

Signed-off-by: Alexey Shvetsov <alexxy@gentoo.org>
Change-Id: Ia330808886b466c4a7d184add56b1ede115c7d0e
---
 lustre/mdt/mdt_handler.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lustre/mdt/mdt_handler.c b/lustre/mdt/mdt_handler.c
index 02dcdb8..38b59a6 100644
--- a/lustre/mdt/mdt_handler.c
+++ b/lustre/mdt/mdt_handler.c
@@ -5505,7 +5505,7 @@ static int mdt_fid2path(const struct lu_env *env, struct mdt_device *mdt,
 {
 	struct mdt_object *obj;
 	struct obd_device *obd = mdt2obd_dev(mdt);
-	int    rc;
+	int    rc = 0;
 	ENTRY;
 
 	CDEBUG(D_IOCTL, "path get "DFID" from "LPU64" #%d\n",
-- 
1.8.1.5

